images_test.go

```diff
@@ -91,4 +91,18 @@ func TestHistoryNames(t *testing.T) {
 	require.Len(t, secondImage.NamesHistory, 2)
 	require.Equal(t, secondImage.NamesHistory[0], "3")
 	require.Equal(t, secondImage.NamesHistory[1], "2")
+
+	// test independent add and remove operations
+	require.Nil(t, store.AddNames(firstImageID, []string{"5"}))
+	firstImage, err = store.Get(firstImageID)
+	require.Nil(t, err)
+	require.Len(t, firstImage.NamesHistory, 5)
```

Comment:
(Even the `.Len` checks can be dropped, the deep equality comparison also implies the same length, AFAIK.)